### DISCUSSION TITLE
Erro na aula da Criação da Calculadora

### DISCUSSION THREAD

**Discussion by BorogeesM** [balta-io/forum#564] [Tira Dúvidas] [answered]
Segui o passo a passo certinho, mas fica mostrando o erro CS8604 Por que? Pesquisei sobre o erro mas não achei muito bem explicado. Tento dar Build e também falha por excesso de tentativas. E Debug também da erro.

![Captura de tela 2024-05-21 021550](https://github.com/balta-io/forum/assets/170201053/93e44f71-b2bd-44a3-b5a0-e03c243472dd)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BorogeesM, provavelmente tem algum processo rodando já... Tenta fechar o visual studio e abrir de novo... qualquer coisa reinicia a máquina.

**Reply by BorogeesM** [reply]
Balta, descobri qual é o erro. Fiquei fuçando aqui e era só parar o debug, dar um build e depois run. Fica um "Warning" no ConsoleReadline mas funciona kkkkkkkkkkk.

Mas balta, brigadão. Tô gostando muito das suas aulas.

**Reply by andrebaltieri** [reply]
hahahahah bem-vindo ao mundo da programação 😂

Show demais que resolveu, meus parabéns pela dedicação e investigação, este é o caminho!
Pra cima 🚀
